arangod/RocksDBEngine/RocksDBKey.h

```diff
@@ -58,6 +58,12 @@ class RocksDBKey {
   RocksDBKey& operator=(RocksDBKey const& other) = delete;
   RocksDBKey& operator=(RocksDBKey&& other) = delete;
 
+  void clear() {
```

Comment:
whats the point of clear? constructDocument overwrites the content

Reply:
Ok, will remove it.